test/coordsTest.js

```diff
@@ -0,0 +1,40 @@
+
+var assert = require('assert');
+var chai = require('chai')
+var expect = chai.expect;
+var coords = require('../scripts/coords.js');
+var _ = require('../scripts/constants.js');
```

Comment:
Rename `_` so it's consistent with `coords.js`. You can also remove the blank line at the top of this file.